lib/template-loader.php

```diff
@@ -319,6 +319,22 @@ function gutenberg_find_template_post_and_parts( $template_type, $template_hiera
 		}
 	}
 
+	// If we haven't found any template post by here, it means that this theme doesn't even come with a fallback
+	// `index.html` block template. We create one so that people that are trying to access the editor are greeted
+	// with a blank page rather than an error.
+	if( ! $current_template_post && ( is_admin() || defined( 'REST_REQUEST' ) ) ) {
+		// 'index' is the ultimate fallback template. If even this template doesn't exist, we create an empty one for it.
```

Comment:
This seems like it has already been covered in the above comment.

Reply:
Good spot, thanks! I'll remove this line.